cockroachcloud/ccloud-get-started.md

```diff
@@ -0,0 +1,434 @@
+
+~~~
+∙∙∙ Creating IP allowlist entry...
+Success! Created IP allowlist entry for
+ network: 1.1.1.1/32
+ cluster: 041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d
+~~~
+
+Use the `ccloud cluster networking allowlist list` command to list the IP allowlists for your cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster networking allowlist list dim-dog
+~~~
+
+~~~
+∙●∙ Retrieving cluster allowlist...
+NETWORK         NAME  UI  SQL
+1.1.1.1/32            ✔   ✔
+~~~
+
+To modify an allowlist entry, use the `ccloud cluster networking allowlist update`. The following command adds a descriptive name to the previously created entry.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster networking allowlist update dim-dog 1.1.1.1/32 --name home
+~~~
+
+~~~
+∙∙● Updating IP allowlist entry...
+Success! Updated IP allowlist entry for
+ network: 1.1.1.1/32
+ cluster: 041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d
+~~~
+
+Rerunning the `allowlist list` command shows the updated entry:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster networking allowlist list dim-dog
+~~~
+
+~~~
+∙∙∙ Retrieving cluster allowlist...
+NETWORK         NAME  UI  SQL
+1.1.1.1/32      home  ✔   ✔
+~~~
+
+To delete an entry, run the `ccloud cluster networking allowlist delete` command.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster networking allowlist delete dim-dog 1.1.1.1/32
+~~~
+
+~~~
+∙∙∙ Deleting IP allowlist entry...
+Success! Deleted IP allowlist entry for
+ network: 1.1.1.1/32
+ cluster: 041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d
+~~~
+
+</section>
+
+## Get a list of all the clusters in your organization
+
+Use the `ccloud cluster list` command to show information about the clusters in your organization. It outputs columns with the cluster name, the cluster ID, the cluster plan, the creation date, the cluster's current state, the cloud provider, and the version of CockroachDB.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster list
+~~~
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+~~~
+∙∙∙ Retrieving clusters...
+NAME                 ID                                    PLAN TYPE        CREATED AT                            STATE                   CLOUD               VERSION
+dim-dog              041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d  PLAN_SERVERLESS  2022-03-20 13:47:40.529531 +0000 UTC  CLUSTER_STATE_CREATED   CLOUD_PROVIDER_GCP  v21.2.4
+...
+~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+~~~
+∙∙∙ Retrieving clusters...
+NAME      ID                                    PLAN TYPE        CREATED AT                            STATE                   CLOUD               VERSION
+dim-dog   041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d  PLAN_DEDICATED   2022-03-22 21:07:35.7177 +0000 UTC    CLUSTER_STATE_CREATING  CLOUD_PROVIDER_GCP  v21.2.4
+...
+~~~
+</section>
+
+## Get information about your cluster using `ccloud cluster info`
```

Comment:
Is there a reason why you include the command in the heading here and below and not in previous headings?